markdown/en/api/ff-asn.api.md

```diff
@@ -35,38 +35,38 @@ ___
 ### Methods
 | Name | Description |
 | ---- | ----------- |
-| **toggle(animated?)** | Toggles the asn-group |
-| **show(animated?)** | Opens the asn-group |
-| **hide(animated?)** | Closes the asn-group |
-| **showDetailedLinksContainer(animated?)** | Opens the detailedLink container. Parameter animated:boolean is optional. |
-| **hideDetailedLinksContainer(animated?)** | Close the detailedLink container. Parameter animated:boolean is optional. |
-| **toggleDetailedLinksContainer(animated?)** | Toggle the detailedLink container. Parameter animated:boolean is optional. |
-| **showHiddenLinksContainer(animated?)** | Opens the hiddenLink container. Parameter animated:boolean is optional. |
-| **hideHiddenLinksContainer(animated?)** | Close the hiddenLink container. Parameter animated:boolean is optional. |
-| **toggleHiddenLinksContainer(animated?)** |Toggle the hiddenLink container. Parameter animated:boolean is optional. |
+| **toggle(animate?: boolean)** | Toggles the asn-group |
+| **show(animate?: boolean)** | Opens the asn-group |
+| **hide(animate?: boolean)** | Closes the asn-group |
+| **showDetailedLinksContainer(animate?: boolean)** | Opens the detailedLink container. Parameter `animate: boolean` is optional. |
```

Comment:
I think we should state _"Parameter `x: type` is optional"_ always, or never, but not sometimes do and sometimes do not. I think the _"?"_  on the left is commonly known, so we could skip that always. I will create a separated issue for that